ui/app/templates/vault/cluster/init.hbs

```diff
@@ -1,20 +1,32 @@
 <SplashPage as |Page|>
   {{#if keyData}}
     <Page.header>
-      <h1 class="title is-4">
-        Vault has been initialized! {{#if (eq keyData.keys.length 1)}}
-        Here is your key.
-        {{else}}
-        Here are your {{pluralize keyData.keys.length "key"}}.
-        {{/if}}
-      </h1>
+      {{#with (or keyData.recovery_keys keyData.keys) as |keyArray|}}
+        <h1 class="title is-4">
+          Vault has been initialized! {{#if (eq keyArray.length 1)}}
```

Comment:
Any harm in moving the conditional to the next line?

Reply:
Nope! I'lld do that